data/json/monsters/zed_fusion.json

```diff
@@ -299,6 +299,7 @@
     "weight": "81500 g",
     "hp": 10,
     "speed": 250,
+    "attack_cost": 200,
```

Comment:
Should undo this one too. Scissorlimbs are inteded to be this fast.

For reference they spawn only within a single end-game location, and are designed to energize  with the rest of the collapsed tower zombies They are the high damage dealing unit  that the rest of the crawlers are designed to screen and slow you for.

This nerf makes the whole monsterline much weaker.